tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -83,92 +140,67 @@ describe(__filename, () => {
   };
 
   it(`calls getClientCompatibility with the add-on's current version if no version is supplied`, () => {
-    const addon = fakeAddon;
-
-    _loadVersions({ slug: addon.slug, versions: [addon.current_version] });
-
     const clientApp = CLIENT_APP_FIREFOX;
-    const _getClientCompatibility = jest.fn().mockReturnValue({
-      compatible: true,
-    });
 
     _dispatchClientMetadata({
       clientApp,
     });
 
-    render({
-      _getClientCompatibility,
-      addon: createInternalAddonWithLang(addon),
-    });
+    renderWithCurrentVersion();
 
     expect(_getClientCompatibility).toHaveBeenCalledWith({
-      addon: createInternalAddonWithLang(addon),
+      addon: createInternalAddonWithLang(fakeAddon),
       clientApp,
-      currentVersion: createInternalVersionWithLang(addon.current_version),
+      currentVersion: createInternalVersionWithLang(fakeAddon.current_version),
       userAgentInfo: store.getState().api.userAgentInfo,
     });
   });
 
   it(`calls getClientCompatibility with a specific version if supplied`, () => {
-    const slug = 'some-slug';
```

Comment:
Further to my comments above, I guess you saw this as a cleanup, but I think the test was better as it was, with `addon` declared explicitly in the test.